atd-etl/cris_import/cris_import.py

```diff
@@ -727,7 +712,7 @@ def group_csvs_into_logical_groups(extracted_archives, dry_run):
     files = os.listdir(str(extracted_archives))
     logical_groups = []
     for file in files:
-        if file.endswith(".xml"):
+        if file.endswith(".xml") or file == "crashReports":
```

Comment:
I'm cherry picking this into this PR from the S3 one because some exports I have floating around (and we all will or already do) which have the PDF containing folder in them, and this checks for that so they don't treated like CSVs.